projects/packages/jetpack-mu-wpcom/src/features/newspack-blocks/index.php

```diff
@@ -86,7 +86,7 @@ function () {
 	);
 
 	add_action(
-		is_admin() ? 'enqueue_block_assets' : 'enqueue_block_editor_assets',
```

Comment:
I wonder what's the historic reason for this.

Reply:
Yes, I asked in https://github.com/Automattic/jetpack/pull/38724 and https://github.com/Automattic/jetpack/pull/38298 🤔

Reply:
Also no other blocks are doing this (otherwise this PR wouldn't entirely remove the packages enqueued).

Reply:
Oops...my memory was wrong, and I just followed the comment below. It seemed something was wrong but I forgot the reason.

https://github.com/Automattic/jetpack/blob/50429029a891294dcec02a3d3e47a4ec4af3f88a/projects/plugins/jetpack/class.jetpack.php#L632-L641

Reply:
Is this going to cause a breakage on P2 @BogdanUngureanu? I just tried adding a timeline and event countdown block in a P2 comment and it worked fine.